aexpect/remote.py

```diff
@@ -372,13 +373,16 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
                     client(specify interface ip)
     :param preferred_authenticaton: Given value of PreferredAuthentications in ssh
     :param user_known_hosts_file: Given value of UserKnownHostsFile in ssh
+    :param additional_params: Additional params for the session, like -X for SSH
     :raise LoginError: If using ipv6 linklocal but not assign a interface that
                        the neighbour attache
     :raise LoginBadClientError: If an unknown client is requested
     :raise: Whatever handle_prompts() raises
     :return: A RemoteSession object.
     """
     verbose = "-vv" if verbose else ""
+    if additional_params:
+        verbose = additional_params + " " + verbose
```

Comment:
Re-using an `verbose` variable for arbitrary params can be misleading, would you please rename the variable? I'd suggest `extra_params`, `custom_args` or any other meaningful variable name which would combine verbose and the extra params.

Reply:
+1

Reply:
Updated as suggested.